xds/src/test/java/io/grpc/xds/PriorityLoadBalancerTest.java

```diff
@@ -214,6 +214,43 @@ public void handleResolvedAddresses() {
     verify(barBalancer0, never()).shutdown();
   }
 
+  @Test
+  public void handleNameResolutionError() {
+    Object fooConfig0 = new Object();
+    PolicySelection fooPolicy0 = new PolicySelection(fooLbProvider, null, fooConfig0);
+    Object fooConfig1 = new Object();
+    PolicySelection fooPolicy1 = new PolicySelection(fooLbProvider, null, fooConfig1);
+
```

Comment:
You could increase the test coverage of this PR by delivering error before delivering any ResolvedAddresses. Not super important, I am ok.